node/src/service.rs

```diff
@@ -402,7 +435,10 @@ pub fn new_full<
                             Ok(d) => d,
                             Err(e) => {
                                 log::warn!("Failed to get distance_threshold: {:?}", e);
-                                tokio::time::sleep(Duration::from_millis(250)).await;
+                                tokio::select! {
+                                    _ = tokio::time::sleep(Duration::from_millis(250)) => {},
```

Comment:
unrelated but wondering if 250ms is too long a time to do nothing at all in a pow node?

Reply:
i think this is an edge case because I don't recall ever seeing this log, it just returns an obligatory Result, so we have to handle that case